templates/patterns/featured_media/pattern-featured-media.html.twig

```diff
@@ -24,7 +24,14 @@
       {% set video_alignment_class = 'order-md-1' %}
     {% endif %}
 
-    <h2 class="mb-4">{{ title }}</h2>
+    {%- if title is not empty -%}
+      {% include '@oe-bcl/heading' with {
+        title: title,
+        title_tag: title_tag|default('h2'),
+        title_link: title_link,
```

Comment:
with my previous comment I meant that the entire line can dissapear, but since you added it to the yml, and I dont see why not, then you can add the default back

Reply:
It should not only be the default as this param is expecting a BCL link object. I added the check if it's exists and merging it accordingly. Also did this for gallery also.